File: mobile/lib/features/forms/domain/entities/service_form_type.dart
class ServiceFormType {
  final int id;
  final String name;
  final String slug;
  final String? description;
  final num feeAmount;
  final bool active;
  final String? createdAt;
  final String? updatedAt;

  const ServiceFormType({
    required this.id,
    required this.name,
    required this.slug,
    this.description,
    required this.feeAmount,
    required this.active,
    this.createdAt,
    this.updatedAt,
  });
}

File: backend/app/Models/ServiceFormType.php
<?php

namespace App\Models;

use App\Traits\BelongsToChurch;
use Database\Factories\ServiceFormTypeFactory;
use Illuminate\Database\Eloquent\Attributes\Fillable;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

#[Fillable([
    'name',
    'slug',
    'description',
    'fee_amount',
    'active',
])]
class ServiceFormType extends Model
{
    /** @use HasFactory<ServiceFormTypeFactory> */
    use BelongsToChurch, HasFactory;

    /**
     * Get the attributes that should be cast.
     *
     * @return array<string, string>
     */
    protected function casts(): array
    {
        return [
            'fee_amount' => 'decimal:2',
            'active' => 'boolean',
        ];
    }

    /**
     * Get the service form applications associated with this form type.
     *
     * @return HasMany<ServiceFormApplication, $this>
     */
    public function applications(): HasMany
    {
        return $this->hasMany(ServiceFormApplication::class);
    }
}
